### PR TITLE
Avatar alert and badge

### PR DESCRIPTION
**Pull request by samyakshah3008**
Hi guys, I have recently integrated three more components in my TripleS UI. Please give your valuable feedback, review it and approve if it looks good to merge 

You can also check out the live link here: 

https://laughing-dubinsky-04453d.netlify.app/ 

Appreciate it!

### CODE REVIEW

File: styles/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+.error{
+    background-color: var(--danger-color);
+    color: var(--secondary-color);
+}
+
+.warning{
+    background-color: var(--warning-color);
+    color: var(--secondary-color);
+
```
Comment (shilpe26) [inline styles/alert/alert.css:28]: remove extra space

File: styles/alert/alert.html [hunk 2]
```diff
@@ -59,8 +59,49 @@ <h3>Components</h3>
...
...https://carbon.now.sh/embed?bg=rgba%28139%2C92%2C246%2C1%29&t=seti&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert%2520primary%2522%253EPrimary%21%2520This%2520is%2520a%2520Primary%2520Alert%2520-%2520Check%2520it%2520now%21%253C%252Fdiv%253E%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert%2520error%2522%253EError%21%2520This%2520is%2520an%2520Error%2520Alert%2520-%2520Check%2520it%2520now%21%253C%252Fdiv%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert%2520warning%2522%253EWarning%21%2520This%2520is%2520a%2520Warning%2520Alert%2520-%2520Check%2520it%2520now%21%253C%252Fdiv%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522alert%2520success%2522%253ESuccess%21%2520This%2520is%2520a%2520Success%2520Alert%2520-%2520Check%2520it%2520now%21%253C%252Fdiv%253E%250A%250A%250A%250A%250A%250A"
```
Comment (shilpe26) [inline styles/alert/alert.html:81]: beautify your files

**Review by MurtazaBagwala07 — APPROVED** [review]
looks good to go

**Review by shilpe26 — APPROVED** [review]
What you need to do is remove all the extra space/lines and apply them to beautify your files, rest you all good to proceed further with the merging

**Pull request merged**